src/python/gem5/components/boards/lupv_board.py

```diff
@@ -94,14 +95,20 @@ def __init__(
 
         self._excep_code = { 'INT_SOFT_SUPER': 1, 'INT_TIMER_SUPER': 5,
                              'INT_TIMER_MACHINE': 7, 'INT_EXT_SUPER': 9,
-                             'INT_EXT_MACHINE': 10 }
+                             'INT_EXT_MACHINE': 11 }
 
         # CLINT
         self.clint = Clint(pio_addr=0x2000000)
 
         # PLIC
         self.pic = Plic(pio_addr=0xc000000)
 
+        # LUPIO IPI
+        self.lupio_ipi = LupioIPI(pio_addr=0x20001000,
+                                  int_type =
+                                  self._excep_code['INT_SOFT_SUPER'],
```

Comment:
Even if it slightly goes over 80 chars, it might be better to merge this line with the previous one. Otherwise, it's kind of hard to read.

Reply:
There is a style checker that doesn't allow us to go over 80 chars so I wasn't sure what to do about this.